tests/samples/__snapshots__/test_api.ambr

```diff
@@ -4,16 +4,16 @@
 # name: TestCreate.test[uvloop-force_choice].1
```

Comment:
Why are there changes in this snapshot file? I don't see why there should be when you are just changing how settings are retrieved.

Reply:
I think these snapshots are actually correct. When the samples data layer was added, the get_settings method in db was used to get the settings since my data layer changes weren't added yet. However, sometimes client.app["settings"] was used to update the settings, which I believe would not be read in get_settings since that checks from the db.